BritneyS/StarWarsDetail
# Refactor

## What you did :question:
- Refactored data model: removed unused elements
- No changes to app behavior

## How you did it :white_check_mark:
- Removed unused class 'Film'
- Removed unnecessary class initializers
- Made data model classes conform to 'Decodable' instead of the less specific 'Codable'


## How to test it :microscope:
- Run the app
- Character names should populate in the table view
- Tap any character's name
- Character Detail View should appear
- Character Detail labels should populate with the property values of the selected character


## Screenshots (if applicable) :camera:
![simulator screen shot - iphone xr - 2018-10-27 at 20 25 06](https://user-images.githubusercontent.com/8409475/47610637-466e9780-da28-11e8-95e9-0470fa2083c8.png)
![simulator screen shot - iphone xr - 2018-10-27 at 20 25 13](https://user-images.githubusercontent.com/8409475/47610638-56867700-da28-11e8-944a-56b53bb69309.png)
